.github/workflows/performance.yaml

```diff
@@ -0,0 +1,145 @@
+name: "Performance Tests"
+
+on:
+  workflow_dispatch:
+
+env:
+  INFLUX_TOKEN: ${{ secrets.INFLUX_TOKEN }}
+  WT_METRICS_DIR: "${{ github.workspace }}/telegraf/metrics"
+
+jobs:
+  test:
+    runs-on: [self-hosted, wind-tunnel]
+    strategy:
+      fail-fast: false
+      matrix:
+        # To run a local test with default configuration, add the scenario name to this array.
+        scenario: [ zome_call_single_value, single_write_many_read, write_read, write_query, local_signals, write_validated ]
+        # To run a local test with additional configuration, add the scenario name and `extra-args` as an `include` item.
+        include:
+          - scenario: dht_sync_lag
+            extra-args: "--agents 2 --behaviour write:1 --behaviour record_lag:1"
+          - scenario: app_install
```

Comment:
The test is for how long app installation takes. So for the smoke test we just care about the test functioning, but when we run the test for results we want know how separately how long each takes between the small and large app

Reply:
Ah okay, nice. Thanks. So it's more about testing how long each takes and maybe even comparing them to one-another instead of testing how long it takes overall. I'll split this up then :-)